apps/task/schema.py

```diff
@@ -41,22 +46,87 @@ class Meta:
 
 
 class TimeEntryType(DjangoObjectType):
+    duration = graphene.String()
+    day_total = graphene.String()
+
     class Meta:
         model = TimeEntry
         fields = '__all__'
 
 
 class TimeEntryTypeList(DjangoObjectType):
+    duration = graphene.String()
+
     class Meta:
         model = TimeEntry
         filterset_class = TimeEntryFilter
 
 
+class SummaryDay(graphene.ObjectType):
+    date = graphene.Date()
+    duration_day = graphene.String()
+    task_list = graphene.List(TimeEntryType)
+
+    def resolve_task_list(root, info, **kwargs):
+        date = root.get('date', None)
+        user = info.context.user
+        queryset = TimeEntry.objects.filter(
+                user=user,
+                date=date
+        )
+        return queryset
+
+
+class SummaryType(graphene.ObjectType):
+    total_hours = graphene.String()
+    total_hours_day = graphene.List(SummaryDay)
+
+
 class Query(object):
     taskgroup = graphene.Field(TaskGroupType)
     taskgroup_list = DjangoFilterListField(TaskGroupListType)
     task = DjangoObjectField(TaskType)
+    task_user = graphene.List(TaskType)
     task_list = DjangoFilterListField(TaskListType)
     timeentry = DjangoObjectField(TimeEntryType)
-    timeentry_list = DjangoFilterListField(TimeEntryTypeList)
-    
\ No newline at end of file
+    summary = graphene.Field(SummaryType)
+
+    def resolve_task_user(root, info):
+        user = info.context.user
+        if not user:
+            return None
+        else:
+            return Task.objects.filter(
+                user=user,
+            )
+
+    def resolve_summary(root, info, **kwargs):
+        date = datetime.date.today()
+        start_week = date - datetime.timedelta(date.weekday())
+        end_week = start_week + datetime.timedelta(6)
+        user = info.context.user
+        if user.is_authenticated:
+            # week total duration
+            queryset_week = TimeEntry.objects.filter(
```

Comment:
queryset_week is not a queryset, so let's use something like this to avoid confusion.
```suggestion
            total_hours_week = TimeEntry.objects.filter(
```